android/src/main/java/com/guichaguri/trackplayer/service/metadata/MetadataManager.java

```diff
@@ -144,7 +144,12 @@ public void updateOptions(Bundle options) {
         builder.setSmallIcon(getIcon(options, "icon", R.drawable.play));
 
         // Update the jump interval
-        jumpInterval = options.getInt("jumpInterval", 15);
+        Object jumpInterval = options.get("jumpInterval");
+        if (jumpInterval instanceof Integer) {
+            this.jumpInterval = options.getInt("jumpInterval", 15);
+        } else if (jumpInterval instanceof Double) {
+            this.jumpInterval = (int) options.getDouble("jumpInterval", 15.0);
+        }
 
         // Update the rating type
         ratingType = options.getInt("ratingType", RatingCompat.RATING_NONE);
```

Comment:
The same issue can happen here too, right? There are many places in the code where I use `getInt`, I think all of them should be fixed too.

Reply:
Yea, that's a good point. Let me just refactor that into a method and replace all `getInt` usages.

Reply:
I have updated the PR for all `getInt` instances.